src/init/omp.ps1

```diff
@@ -23,6 +23,19 @@ function global:Set-PoshGitStatus {
     }
 }
 
+function global:Set-AzSubscription {
```

Comment:
Looking at that we now also have `Set-PoshGitStatus` I would maybe rename that one to `Initialize-ModuleSupport` and add this logic there as well. That way we have 1 spot for those items and expand as we see fit.

Reply:
Sure, I can do that

Reply:
done!